frontend/app/src/services/rest/RestService.ts

```diff
@@ -16,3 +18,8 @@ export function uploadPlayerCode(
 export function getStatusV0(): Promise<Response> {
 	return fetch(`${import.meta.env.VITE_REACT_APP_BACKEND_URL}/v0/status`);
 }
+
+export function getLinterLintsV1(code: string): Promise<Linterlint[]> {
+	console.debug("fetching lints for code:, ", code);
```

Comment:
1. this function is not yet implemented, so this log will be removed once it is.
2.it is a debug-level log, it is just nice to see that apparently the intended behaviour is happening
3. there a pretty similiar debug logs on the components you implemented that use some kind of polling, so it would actually be inconsistent to remove this here (or remove at everywhere else too ?)

Reply:
Then I think we should remove them from there aswell, or come together and discuss a general concept for how, where and what we log.

I would prefer the first option atm.